tests/code-splitting/subtests/between-workers/webpack/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-result: fail
+result: pass
 ---
 
-Webpack will create and load duplicate copies of dependencies even if they're shared by each worker bundle.
+Using `new Worker(new URL("./worker", import.meta.url))` will create a worker that is able to share dependencies with the main bundle (and other worker bundles).
```

Comment:
Interesting. I didn't know that `importScripts` is not available in module workers.

Yes for "module" output in webpack we need a need chunk loading mechanism, which uses `import()`. For Workers this would actually an improvement since `import()` doesn't block compared to `importScripts`.

Reply:
If it doesn't work for module workers, I guess it's a partial pass? However, that seems a bit unfair to webpack. @developit WDYT to splitting this test into one for classic workers and one for module workers, where webpack would pass one and fail the other? I'm happy to do the work to split these up.

Reply:
I think we should call this a pass for `between-workers`, but then mark Webpack as "partial" for `between-new-worker-types`.  Webpack assumes classic workers as output for its built-in worker support, which is very reasonable given browser support and should be a "pass". It's more the "new/additional types of contexts" where the importScripts assumption falls over, and I think we can track the status better in the other test.

Reply:
I think we should call this a pass for `between-workers`, but then mark Webpack as "partial" for `between-new-worker-types`.  Webpack assumes classic workers as output for its built-in worker support, which is very reasonable given browser support and should be a "pass". It's more the "new/additional types of contexts" where the importScripts assumption falls over, and I think we can track the status better in the other test.

Reply:
Hm ok, let's go with 'pass' here. Although I'm worried it's a little misleading because there are worker types it can't handle. I'm less bothered about module workers, and more bothered about worklets, where there isn't an alternative for the developer.